.github/workflows/percy_snapshots.yml

```diff
@@ -0,0 +1,30 @@
+name: Percy Snapshots
+
+on: 
+  push:
+    branches-ignore: dev/*
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build
+    - name: Percy Snapshots
+      uses: percy/exec-action@v0.3.1
+      with:
+        command: npx testcafe "chrome:headless" tests/acceptance/percysnapshots.js
```

Comment:
Different browsers have different user agent stylings. Is it possible to generate snapshots for a build in additional browsers?

Reply:
percy takes care of that automatically, though only firefox and chrome are supported. you only need to run testcafe in one browser. running in multiple will actually have percy complain about "multiple snapshots with the same name in the same build"

Reply:
here's an example of percy complaining when you run it (manually) on multiple browsers https://app.circleci.com/pipelines/github/yext/answers/4773/workflows/1278c236-94c8-49d6-bbb7-3ce0453d6ce0/jobs/13226